diff --git a/src/export/ExportTemplateLoader.cpp b/src/export/ExportTemplateLoader.cpp
index e08abcd0..07c068d5 100644
--- a/src/export/ExportTemplateLoader.cpp
+++ b/src/export/ExportTemplateLoader.cpp
@@ -6,8 +6,8 @@
 #include <QNetworkReply>
 #include <QXmlStreamReader>
 
-#include "quazip/quazip/quazip.h"
-#include "quazip/quazip/quazipfile.h"
+#include "quazip5/quazip.h"
+#include "quazip5/quazipfile.h"
 #include "settings/Settings.h"
 
 #include "data/Storage.h"
