diff --git a/src/tvShows/TvShowUpdater.cpp b/src/tvShows/TvShowUpdater.cpp
index eea223e5..1151c52d 100644
--- a/src/tvShows/TvShowUpdater.cpp
+++ b/src/tvShows/TvShowUpdater.cpp
@@ -10,8 +10,8 @@
 #include "globals/Globals.h"
 #include "globals/Manager.h"
 #include "notifications/NotificationBox.h"
-#include "quazip/quazip/quazip.h"
-#include "quazip/quazip/quazipfile.h"
+#include "quazip5/quazip.h"
+#include "quazip5/quazipfile.h"
 #include "scrapers/TheTvDb.h"
 
 TvShowUpdater::TvShowUpdater(QObject *parent) : QObject(parent), m_tvdb{nullptr}
