diff --git a/CMakeLists.txt b/CMakeLists.txt
index e61a5b9..9d20efb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -244,7 +244,13 @@ include(FindPkgConfig)
 pkg_search_module(UCHARDET REQUIRED uchardet)
 
 include(translations/translatorsinfo.cmake)
-create_translatorsinfo_qrc(TRANSLATORS_INFO_QRC translations)
+add_custom_target("translators")
+add_custom_command(
+       TARGET "translators"
+       PRE_BUILD
+	BYPRODUCTS ${TRANSLATORS_INFO_FILE}
+	COMMAND create_translatorsinfo_qrc(TRANSLATORS_INFO_QRC translations))
+
 set(LIBRARIES ${LIBRARIES} ${UCHARDET_LIBRARIES})
 include_directories(${UCHARDET_INCLUDE_DIRS})
 link_directories(${UCHARDET_LIBRARY_DIRS})
diff --git a/translations/translatorsinfo.cmake b/translations/translatorsinfo.cmake
index b439a05..b586451 100644
--- a/translations/translatorsinfo.cmake
+++ b/translations/translatorsinfo.cmake
@@ -23,13 +23,11 @@
  #
  # END_COMMON_COPYRIGHT_HEADER
 
+set(TRANSLATORS_INFO_FILE ${CMAKE_CURRENT_BINARY_DIR}/translators.info)
 
 FUNCTION(create_translatorsinfo_qrc _qrcFile _DIR)
     get_filename_component(srcDir ${_DIR}/ ABSOLUTE)
 
-
-    set(TRANSLATORS_INFO_FILE ${CMAKE_CURRENT_BINARY_DIR}/translators.info)
-
     file(GLOB IN_FILES ${srcDir}/translators_*.info)
     file(WRITE ${TRANSLATORS_INFO_FILE} "")
 
